configs/linux/MinimalAgent/Ubuntu/Ubuntu.Dockerfile

```diff
@@ -52,30 +53,23 @@ ENV CONFIG_FILE=/data/teamcity_agent/conf/buildAgent.properties \
 LABEL dockerImage.teamcity.version="latest" \
       dockerImage.teamcity.buildNumber="latest"
 
-COPY run-agent.sh /run-agent.sh
-COPY run-agent-services.sh /run-services.sh
-COPY TeamCity/buildAgent /opt/buildagent
+COPY --chown=buildagent:buildagent run-agent.sh /run-agent.sh
+COPY --chown=buildagent:buildagent run-agent-services.sh /run-services.sh
+COPY --chown=buildagent:buildagent TeamCity/buildAgent /opt/buildagent
 
-RUN apt-get update && \
-    useradd -m buildagent && \
-    chmod +x /opt/buildagent/bin/*.sh && \
+RUN chmod +x /opt/buildagent/bin/*.sh && \
```

Comment:
By combining those two `RUN` commands, the number of layers in the final image will reduce by one.